components/section/Project.jsx

```diff
@@ -16,81 +16,148 @@ export default function Project() {
   ];
 
   const images = [
-    { image: "https://raw.githubusercontent.com/TheGymRwanda/unit8-red/dev/public/assets/images/dacade.webp", text: "Peer-to-Peer learning platform" },
-    { image: "https://raw.githubusercontent.com/TheGymRwanda/unit8-red/dev/public/assets/images/bitlipa.webp", text: "Peer-to-Peer learning platform" },
-    { image: "https://raw.githubusercontent.com/TheGymRwanda/unit8-red/dev/public/assets/images/symplifi.webp", text: "Peer-to-Peer learning platform" },
-    { image: "https://raw.githubusercontent.com/TheGymRwanda/unit8-red/dev/public/assets/images/utu.webp", text: "Peer-to-Peer learning platform" },
-    { image: "https://raw.githubusercontent.com/TheGymRwanda/unit8-red/dev/public/assets/images/neueux.webp", text: "Peer-to-Peer learning platform" },
-    { image: "https://raw.githubusercontent.com/TheGymRwanda/unit8-red/dev/public/assets/images/lab3.webp", text: "Peer-to-Peer learning platform" },
-    { image: "https://raw.githubusercontent.com/TheGymRwanda/unit8-red/dev/public/assets/images/even.webp", text: "Peer-to-Peer learning platform" },
+    {
+      image:
```

Comment:
You did a great job, the only simple thing to change is to remove white space.  try to prettier to make it perfect.